tekton-catalog/pipeline-loops/pkg/apis/pipelineloop/v1alpha1/pipelineloop_defaults.go

```diff
@@ -31,4 +31,7 @@ func (tl *PipelineLoop) SetDefaults(ctx context.Context) {
 
 // SetDefaults set any defaults for the PipelineLoop spec
 func (tls *PipelineLoopSpec) SetDefaults(ctx context.Context) {
+	if tls.Parallelism == 0 {
+		tls.Parallelism = 1
```

Comment:
Can we set the default Parallelism number as a environment variable? This way the admin can change it based on the cluster size.

Reply:
Do you mean to make it an environment variable for webhook? Though user can set it purposely but I agree it's always good to have a suitable default value, I can do that.